kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/KuraErrorCode.java

```diff
@@ -54,5 +55,10 @@
 	/**
 	 * @since {@link org.eclipse.kura} 1.2.0
 	 */
-	INVALID_PARAMETER
+	INVALID_PARAMETER,
+	/**
+	 * Error executing process
+	 * @since {@link org.eclipse.kura} 3.0.0
```

Comment:
Maybe I'm wrong but I think that @cdealti for INVALID_PARAMETER and OS_COMMAND_ERROR with {@link org.eclipse.kura} referred the version for the org.eclipse.kura package inside the org.eclipse.kura.api project.